src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -28,6 +28,8 @@ public Actor add(Actor actor) {
                 transaction.rollback();
             }
             throw new DataProcessingException("Can not add actor to DB", e);
+        } finally {
+            session.close();
```

Comment:
man, please, pay more attention, you did this in other DAO classes 
its a piece of your code, be consistent, add null check here also
![image](https://github.com/mate-academy/hibernate-relations-hw/assets/48316721/9a4ddd43-a594-4b9f-8c00-656470bec0b0)

Reply:
ohh, sorry